src/lib.rs

```diff
@@ -14,9 +14,21 @@ mod signatures;
 pub mod gadgets;
 
 #[deprecated(note = "Please use DoubleSignature instead")]
-pub type Proof = signatures::double::Signature;
```

Comment:
Do we may need that for backwards compatibility? (see deprecated macro above)

Reply:
I thought it's nice to keep it in for this release.